src/components/structures/MatrixChat.js

```diff
@@ -506,18 +506,11 @@ module.exports = React.createClass({
             page_type: this.PageTypes.RoomView,
             thirdPartyInvite: thirdPartyInvite,
             roomOobData: oob_data,
+            currentRoomAlias: roomAlias,
```

Comment:
could you add `currentRoomAlias` and `currentRoomId` to `getInitialState`, along with some comments about what they are and how they interact?